Increase the timeout while waiting for the type attribute.
This is needed for some fibre channel hardware (Sun StorEdge
A5000), that only begin to spin up the disk much later than
this. Patch has no side effects.

Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>

--- etc/hotplug/scsi.agent.orig	2004-09-20 14:29:54.000000000 -0700
+++ etc/hotplug/scsi.agent	2006-03-14 18:37:43.000000000 -0800
@@ -22,7 +22,7 @@
 
     # Possibly sleep here to try and avoid races with scsi attributes and block
     # devices
-    count=10
+    count=120
     while [ ! -f $TYPE_ATTR ] && [ $count -gt 0 ]
     do
 	# We seem to always hit this now, so don't output any message.
